scale/docs/rest/v6/scheduler.rst

```diff
@@ -0,0 +1,100 @@
+
+.. _rest_v6_scheduler:
+
+v6 Scheduler Services
+==================
+
+These services provide access to information about the scheduler.
+There should be one and only one scheduler entry in the database and it is used to store global data like paused status.
```

Comment:
I would remove this line since it reveals system details not necessary for an API caller.